truthbeam/go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/complytime/complybeacon/truthbeam
 
-go 1.24.5
+go 1.24.0
```

Comment:
Any particular reason to downgrade? There are security fixes after `1.24.0`.

Reply:
Good catch. Thank you. That's must have changed when I ran `go mod tidy`.